### DISCUSSION TITLE
NuxtLink is updating route in nuxt 3 app, but not rendering components

### DISCUSSION THREAD

**Discussion by svJariwala** [nuxt/nuxt#16976] [Questions]
https://stackoverflow.com/q/73541869/19878692

**Comment by TheAlexLichter** [comment]
Please provide a reproduction, e.g. via StackBlitz or CodeSandbox. Read more [here](http://v3.nuxtjs.org/community/reporting-bugs#create-a-minimal-reproduction)

**Reply by svJariwala** [reply]
Okay

**Reply by svJariwala** [reply]
I fixed this thank you that's my mistake

**Reply by TheAlexLichter** [reply]
Feel free to share how so it can help others too 😋
